vars/withNodejsEnv.txt

```diff
@@ -0,0 +1,10 @@
+Install Node.js with NVM and run some command in a pre-configured environment multiplatform. For such
+it's recommended to use the `cmd` step.
+
+```
+  withNodejsEnv(version: '14.17.5'){
+    cmd(label: 'Node version', script: 'node --version')
+  }
```

Comment:
```suggestion
  withNodejsEnv(version: '14.17.5'){
    cmd(label: 'Node version', script: 'node --version')
  }
  
  withNodejsEnv(version: readFile(file: '.nvmrc').trim()){
    cmd(label: 'Node version', script: 'node --version')
  }
```

Reply:
`version` already supports the `readFile '.nvmrc'` see https://github.com/elastic/apm-pipeline-library/pull/1402/files#diff-425e2866e7902441a31cb5a9e3a92a73ea68ee28c100cd097b2ab70d7ae50dfbR25-R28